tex/quantum/vectors.tex

```diff
@@ -476,12 +479,14 @@ \section{Entanglement}
 	\]
 	As for the paradox: what happens if you multiply all these measurements together?
 	\begin{hint}
-		$-1$, $1$, $1$, $1$.
+		$1$, $1$, $1$, $-1$ respectively.
 		When we multiply them all together,
 		we get that $\id^A \otimes \id^B \otimes \id^C$
 		has measurement $-1$, which is the paradox.
 		What this means is that the values of the measurements
-		are created when we make the observation,
-		and not prepared in advance.
+		can't be prepared in advance independently. 
+		In other words, this contradicts certain local hidden-variable theories. 
+
+		This result is worth a Nobel prize.
```

Comment:
maybe we can be a little less snarky lol

Reply:
I don't consider this that snarky, so I'm unable to think of how to change it. Any suggestions?